test/Microsoft.AspNetCore.Razor.Test/CodeGenerators/CSharpTagHelperRenderingTest.cs

```diff
@@ -1788,9 +1845,9 @@ public static TheoryData RuntimeTimeTagHelperTestData
                     { "AttributeTargetingTagHelpers", null, AttributeTargetingTagHelperDescriptors },
                     { "PrefixedAttributeTagHelpers", null, PrefixedAttributeTagHelperDescriptors },
                     {
-                        "PrefixedAttributeTagHelpers",
-                        "PrefixedAttributeTagHelpers.Reversed",
-                        PrefixedAttributeTagHelperDescriptors.Reverse()
+                       "PrefixedAttributeTagHelpers",
+                       "PrefixedAttributeTagHelpers.Reversed",
+                       PrefixedAttributeTagHelperDescriptors.Reverse()
```

Comment:
Well something is wrong since these lines are now indented 3 spaces w.r.t. the curly brace.